WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -830,9 +830,11 @@ public void onStart() {
 
     @Override
     public void setTitle(@NonNull final String title) {
-        mToolbarTitle = title;
-        if (mToolbar != null) {
-            mToolbar.setTitle(title);
+        if (isAdded()) {
+            mToolbarTitle = title.isEmpty() ? getString(R.string.wordpress) : title;
```

Comment:
So sorry to delay this PR even further @manisha-A but, I just noticed that we don't make sure to avoid passing `null` as a parameter to `setTitle()`. That means that `title.isEmpty()` will crash if null is passed. Since the method is overridden and marked with `@NonNull`, let's add null guards to its call sites. Can you please update the PR to add those null guards? Let me know if this is unclear in any way. Thanks!

Reply:
Hi @hypest, Do you mean while setting `mToolbarTitle`  we should be checking `title == ""` instead of null check as it can never be null?

Reply:
Hey @manisha-A , sorry it wasn't clear. I mean that we should actually check for null.

There are places that call this method but the `title` they pass is not guaranteed to be non-null ([example](https://github.com/wordpress-mobile/WordPress-Android/blob/78db0202950f3a865e8cb3f63872f9495134d43f/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java#L600)). So, even though the method's param is marked with `@NonNull`, I think it's better if we check for null.

Notice that the previous code didn't need the check because the value was just passed to `mToolbar.setTitle(title)` which internally does the check. But now, `title.isEmpty()` can fail when title is null.

Let me know if it's clearer now, thanks!